src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -636,7 +636,7 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                       id='withdrawnQuantity'
                       onChange={(value: unknown) => setWithdrawnQuantity(value as number)}
                       type='text'
-                      value={numericFormatter.format(withdrawnQuantity)}
+                      value={withdrawnQuantity}
```

Comment:
this was a bugfix... format was intended for the total ready quantity

Reply:
So the rule is to format on "non editable" fields?

Reply:
Rule is to not format what the user is typing. Other display values should be formatted I think that’s the plan. You had also called it out in an earlier PR. We need to do this in all the accessions views as well.